.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 'Bug report'
+labels: Bug
+assignees: bep
```

Comment:
Remove the labels and assignees. An issue isn't a bug just because some user says so, and these issues will never be fully automatic. Someone needs to look at them.